css/style.css

```diff
@@ -0,0 +1,291 @@
+* {
+   box-sizing: border-box;
+}
+
+.body {
+   max-width: 1440px;
+   margin: 0 auto;
+   font-family: "Montserrat", sans-serif;
+   color: #000;
+ }
+
+ .header {
+   margin: 1.7% 3.8% 0 2.8%;
+   display: flex;
+   justify-content: space-between;
+   align-items: center;
+ }
+
+ .header__img {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)